firstvoices/backend/tests/test_apis/test_category_api.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+import json
+from rest_framework.test import APIClient
+
+from backend.tests.test_apis.base_api_test import BaseSiteContentApiTest
+from backend.tests.factories import get_app_admin, SiteFactory, ParentCategoryFactory, ChildCategoryFactory
+from backend.models.constants import AppRole
+
+
+class TestCategoryEndpoints(BaseSiteContentApiTest):
+    """
+    End-to-end tests that the category endpoints have the expected behaviour.
+    """
+    API_LIST_VIEW = "api:category-list"
+    API_DETAIL_VIEW = "api:category-detail"
+
+    def setup_method(self):
+        self.client = APIClient()
+        self.user = get_app_admin(AppRole.STAFF)
+        self.client.force_authenticate(user=self.user)
+        self.site = SiteFactory.create()
+
```

Comment:
I think these need some tests about permissions, but let's leave it out for now. I have an idea about how to put most of those things in the base test class